spec/controllers/api/v1/sandwiches_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::SandwichesController, type: :controller do
+  include Devise::Test::ControllerHelpers
+
+  let!(:sandwich_one){ Sandwich.create(id:1, name:"reuban", description: "delicious", image_url: "www.google.com" )}
+  let!(:sandwich_two){ Sandwich.create(id:2, name:"grilled cheese", description: "gooey", image_url: "www.yahoo.com" )}
+
+  describe "GET#index" do
+    before do
+      get :index
+
+    end
+
+
+
+    it "returns http success" do
```

Comment:
Watch your spacing. Remove line breaks from lines 12, 14, and 15. Also your ends on line 43 and 44 seem a bit scattered. On line 44 the `end` should only be indented 3 spaces to line up with `describe` on line 9. And on line 43, the `end` should be indented 5 spaces to line up with the `it` on line 21.

Reply:
I fixed this, adding a new pull request with the changes